### PR TITLE
fix x-rays visible when disabled

### LINKED ISSUE #26571: X-Ray Features cannot be disabled
**Describe the bug**
Disabling X-Ray features in 'Admin Settings > General' does not remove exploration options from homepage.

**Information about your Metabase Installation:**

You can get this information by going to Admin -> Troubleshooting.

- Your browser and the version: Chrome 106.0
- Your operating system: Ubuntu 20.04
- Metabase version: v0.44.6
- Metabase hosting environment: Docker
- Metabase internal database: Postgres

**Severity**
Since resources are limited, explorations being visible would allow less knowledgeable users to consume resources carelessly.

### PR DESCRIPTION
**Pull request by alxnddr**
Closes https://github.com/metabase/metabase/issues/26571

### Description

When x-rays disabled in settings we should not render them even for admins on the home page.

### How to verify

1. Run a completely new instance
2. Go to Admin Settings -> General -> Disable x-rays
3. Go to the home page and ensure there is no x-rays

### Checklist

- [x] Tests have been added/updated to cover changes in this PR

### CODE REVIEW

**Comment by nemanjaglumac** [comment]
@alxnddr when disabled?

File: e2e/test/scenarios/dashboard/x-rays.cy.spec.js [hunk 1]
```diff
@@ -21,11 +21,10 @@ describe("scenarios > x-rays", () => {
 
   const XRAY_DATASETS = 11; // enough to load most questions
 
-  it.skip("should not display x-rays if the feature is disabled in admin settings (metabase#26571)", () => {
+  it("should not display x-rays if the feature is disabled in admin settings (metabase#26571)", () => {
     cy.request("PUT", "api/setting/enable-xrays", { value: false });
 
     cy.visit("/");
-    cy.findByText("Metabase tips");
```
Comment (nemanjaglumac) [inline e2e/test/scenarios/dashboard/x-rays.cy.spec.js:28]: Was this check creating problems?

Comment (alxnddr) [reply e2e/test/scenarios/dashboard/x-rays.cy.spec.js:28]: When a user does not have any recent or popular items we don't show "Metabase tips"

**Review by npfitz — APPROVED** [review]

**Comment by npfitz** [comment]
Looks good to me!

**Pull request merged**